### DISCUSSION TITLE
Support search exclusion filters in GitHub Notification search

### DISCUSSION THREAD

**Discussion by cameronbrill** [community/community#111363] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Currently, Github Notifications search [does not support exclusion filters](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox#custom-filter-limitations:~:text=Search%20exclusion%20using%20NOT%20or%20%2DQUALIFIER.). This is unfortunate, as my job does work in a monorepo, which results in the vast majority of my notifications resulting from a single GitHub repo, often drowning out all other notifications. I have notifications coming from dozens of GitHub repos, many of which I haven't contributed to in months or years, so adding `repo:xyz` for each repo I'd like to see notifications from is not a viable options, effectively making the GitHub notifications tab useless.

**Comment by asasine** [comment]
Yes please! I use the same GitHub account for work and personal and work notifications drown out personal notifications, making it difficult to utilize notifications reliably for other projects.

> Custom filters do not currently support:
> ...
> Search [exclusion](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#exclude-certain-results) using NOT or -QUALIFIER

https://docs.github.com/en/subscriptions-and-notifications/reference/inbox-filters#custom-filter-limitations
